### ISSUE TITLE
Logs are colorized everywhere

### ISSUE THREAD

**Issue by aldofunes** [winstonjs/logform#60]
I am having some trouble with the colorizing of logs. Specifically, when logging to non-tty outputs.

There is an override in [colorize.js:9](https://github.com/winstonjs/logform/blob/master/colorize.js#L9) that prevents the `colors` package to enable or disable when supported.

First of all, I would like to know why this decision was made, and if it can be removed.

I have made a hack as a workaround. Here it is, in case anyone wants to know how to do it:

```javascript
    format: format.combine(
      LOGGER_COLORS === 'true' ? format.colorize({ all: true }) : format(i => i)(),
      format.printf(({
        label = 'default,
        namespace = 'default',
        level = 'debug',
        message,
        timestamp,
      }) => {
        let string = `${label}:${namespace} ${level}: ${message}`;

        if (LOGGER_CONSOLE_TIMESTAMP === 'true') {
          string = `${timestamp} ${string}`;
        }

        return string;
      }),
    ),
```

**Comment by DABH** [comment]
I agree with your concerns -- it would probably be better if colorizer accepted a `force` option which would set `colors.enabled=true;` if `true`, but otherwise `winston` doesn't mess with `colors`.  That way by default colorizer will do whatever `colors` thinks is the right thing, which is usually correct :) . Want to make a quick PR for that change?

**Comment by maxsatula** [comment]
Have the same problem and the same question "why".
However, I prefer to do a different workaround to delegate `colors` module to decide rather than provide environment variables.

Here, if anyone finds my (simplified) example useful:

```js
'use strict';
// IMPORTANT! colors.enabled value must be captured before Winston imported!
const colorsEnabled = require('colors/safe').enabled;
const winston = require('winston');

winston.add(
    new winston.transports.Console({
        format: winston.format.combine(
            colorsEnabled // the rest is pretty much the same idea as @aldofunes
                ? winston.format.colorize({ all: false })
                : winston.format(i => i)(),
            winston.format.simple()
        )
    })
);

winston.warn('Some warning');
```

@DABH, once I have a chance to do PR, what would be the `force` desirable default? Naturally I would expect `false` but not sure how bad that is from backwards compatibility behavior standpoint.

**Comment by maxsatula** [comment]
Maybe it's worth to note that unless colors are forced, an app running in a docker container will not produce a colorized output if `-t, --tty` is not specified in `docker run` command
